packages/postgres-database/src/simcore_postgres_database/models/file_meta_data.py

```diff
@@ -56,4 +68,5 @@
         doc="SHA256 checksum of the file content",
         index=True,
     ),
+    ###
```

Comment:
maybe not needed

Reply:
the `###` comment or the change of the model? :D